### PR TITLE
Split packages

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
@conda-forge-admin, please rerender

**Comment by scopatz** [comment]
Thanks for pushing on this @isuruf!

**Comment by isuruf** [comment]
@mingwandroid, have you seen the following error before?
```
[  7%] Building C object LAPACKE/example/CMakeFiles/xexample_DGELS_rowmajor.dir/example_DGELS_rowmajor.c.obj
C:\bld\blas-split_1546939921972\work\LAPACKE\example\example_DGELS_rowmajor.c:61:19: fatal error: "/stdio.h: Invalid argument
compilation terminated.
mingw32-make[2]: *** [LAPACKE/example/CMakeFiles/xexample_DGELS_rowmajor.dir/example_DGELS_rowmajor.c.obj] Error 1
LAPACKE\example\CMakeFiles\xexample_DGELS_rowmajor.dir\build.make:61: recipe for target 'LAPACKE/example/CMakeFiles/xexample_DGELS_rowmajor.dir/example_DGELS_rowmajor.c.obj' failed
mingw32-make[1]: *** [LAPACKE/example/CMakeFiles/xexample_DGELS_rowmajor.dir/all] Error 2
mingw32-make[1]: *** Waiting for unfinished jobs....
CMakeFiles\Makefile2:2074: recipe for target 'LAPACKE/example/CMakeFiles/xexample_DGELS_rowmajor.dir/all' failed
[  7%] Linking Fortran executable ..\..\bin\xblat3z.exe
[  7%] Built target xblat3z
mingw32-make: *** [all] Error 2
Makefile:161: recipe for target 'all' failed
```

**Comment by mingwandroid** [comment]
@isuruf, no I haven't seen this, is it still a problem? If so, can you add `VERBOSE=1` to your build/make command?

**Comment by isuruf** [comment]
Last appveyor build log is a verbose build and fails with,
```
cd /d %SRC_DIR%\build\LAPACKE\example && %BUILD_PREFIX%\Library\mingw-w64\bin\gcc.exe   -O3 -DNDEBUG   -o CMakeFiles\xexample_DGELS_rowmajor.dir\example_DGELS_rowmajor.c.obj   -c %SRC_DIR%\LAPACKE\example\example_DGELS_rowmajor.c
C:\bld\blas-split_1546965379820\work\LAPACKE\example\example_DGELS_rowmajor.c:61:19: fatal error: "/stdio.h: Invalid argument
```

**Comment by mingwandroid** [comment]
@isuruf, can you use `MSYS Makefiles`, `m2-make` and `make` instead? You may need to convert a bunch of vars from Windows form to posix form using cygpath. A good few feedstocks do this already but one example can be seen at this (mostly deprecated) URL: https://github.com/conda/conda-recipes/blob/master/msys2/m2w64-muparser/bld.bat

**Comment by isuruf** [comment]
Thanks @mingwandroid. Error went away when m2-make and MSYS Makefiles was used

**Comment by mariusvniekerk** [comment]
@conda-forge-admin, please rerender

**Comment by isuruf** [comment]
@conda-forge/core, has anyone seen the error in ppc64le before? Looks like a conda-build bug, but it doesn't happen in any other arch.

**Comment by isuruf** [comment]
nvm, conda-build is old for ppc64le

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,31 +1,190 @@
...
+      build:
+        - {{ compiler('c') }}                 # [unix]
+        - {{ compiler('fortran') }}           # [unix]
+        - {{ compiler('m2w64_c') }}           # [win]
+        - {{ compiler('m2w64_fortran') }}     # [win]
+        - cmake
+        - m2-make   # [win]
+        - posix     # [win]
+      host:
+      # Building with blis fails due to a conda-build bug
```
Comment (h-vetinari) [inline recipe/meta.yaml]: Has this been resolved? By the sound of [this comment](https://github.com/conda-forge/blas-feedstock/pull/16#issuecomment-471365135), it should be fine now? @isuruf

**Pull request merged**